service/config/config.go

```diff
@@ -100,8 +101,8 @@ type ReplacementVars struct {
 
 // LoadConfig parses config.
 func LoadConfig(path string, vars ReplacementVars) (conf *Config, err error) {
-	if _, err = os.Stat(path); os.IsNotExist(err) {
-		return nil, fmt.Errorf("The configuration file does not exist. Please place the file here: %s", path)
+	if !osutils.FileExists(path) {
+		return nil, fmt.Errorf("The conf file does not exist. Please configuration here: %s", path)
```

Comment:
return nil, fmt.Errorf("The conf file does not exist. Please configuration here: %s", path)
->
return nil, fmt.Errorf("The conf file does not exist. Please put the configuration file here: %s", path)

Reply:
will do